src/azul/indexer.py

```diff
@@ -147,3 +144,13 @@ def on_error(doc_id, error):
             log.warning('%s.%s: conflicts: %r', bundle_uuid, bundle_version, conflict_documents)
             raise RuntimeError('%s.%s: Failed to index bundle. Failures: %i, conflicts: %i.' %
                                (bundle_uuid, bundle_version, len(errored_documents), len(conflict_documents)))
+
+    def define_indexes(self):
```

Comment:
Why do we need to make this into a function? Do we need a separated `es_client`?